### DISCUSSION TITLE
Digits der Wasseruhr immer 1 Stelle zu niedrig

### DISCUSSION THREAD

**Discussion by aherrlich** [jomjol/AI-on-the-edge-device#2890] [Q&A] [answered]
Nach dem Update auf Version 15.5. liest die Wasseruhr die Digits immer 1 Stelle zu niedrig aus, also statt 184 (was korrekt wäre) wird 183 ausgelesen. Habe gerade das Update 15.6. ausprobiert, mit dem gleichen Resultat. Die analogen Zeiger sind korrekt. Beim Gaszähler tritt das nicht auf.

**Comment by rgiese11** [comment]
Da bist du nicht der einzige. Dasselbe Problem tritt bei mir auch auf. Bin zurück auf 15.4.0. Da läuft alles wieder.

**Comment by aherrlich** [comment, marked as answer]
Vielen Dank für die schnelle Antwort. Bin auch auf die alte Version zurück.
